_site/index.html

```diff
@@ -628,6 +722,21 @@ <h3>T-Z</h3>
 
 		</section>
 
+    <section>
+      <p class="word__title" id="transgender">
+        Transgender
+        <span class="word__speech">
+          adj
+        </span>
+      </p>
+      
+      <p class="word__definition">
```

Comment:
`      <p class="word__definition">
       of a gender does not match the gender they were assigned at birth.<br><br>`

Reply:
I'm reframing it in this way to be grammatically correct, since the inclusion of "someone who makes this a noun." 🤓

Reply:
Makes sense! Done.